### ISSUE TITLE
Magnifying glass requests game profile service too often, spamming logs

### ISSUE THREAD

**Issue by liach** [Railcraft/Railcraft#1450]
Example stacktrace:
```
[21:24:44] [Server thread/WARN] [mojang/YggdrasilMinecraftSessionService]: Couldn't look up profile properties for com.mojang.authlib.GameProfile@748f38e6[id=2ecec10b-3fca-3e44-9b51-4cca88e1c207,name=Player598,properties={},legacy=false]
com.mojang.authlib.exceptions.AuthenticationException: The client has sent too many requests within a certain amount of time
	at com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService.makeRequest(YggdrasilAuthenticationService.java:79) ~[YggdrasilAuthenticationService.class:?]
	at com.mojang.authlib.yggdrasil.YggdrasilMinecraftSessionService.fillGameProfile(YggdrasilMinecraftSessionService.java:180) [YggdrasilMinecraftSessionService.class:?]
	at com.mojang.authlib.yggdrasil.YggdrasilMinecraftSessionService.fillProfileProperties(YggdrasilMinecraftSessionService.java:173) [YggdrasilMinecraftSessionService.class:?]
	at net.minecraft.tileentity.TileEntitySkull.updateGameprofile(TileEntitySkull.java:166) [TileEntitySkull.class:?]
	at mods.railcraft.common.plugins.forge.ChatPlugin.modifyArgs(ChatPlugin.java:81) [ChatPlugin.class:?]
	at mods.railcraft.common.plugins.forge.ChatPlugin.sendLocalizedChatFromServer(ChatPlugin.java:64) [ChatPlugin.class:?]
	at mods.railcraft.common.items.ItemMagnifyingGlass.onEntityInteract(ItemMagnifyingGlass.java:84) [ItemMagnifyingGlass.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_19_ItemMagnifyingGlass_onEntityInteract_EntityInteract.invoke(.dynamic) [?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) [ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) [EventBus.class:?]
	at net.minecraftforge.common.ForgeHooks.onInteractEntity(ForgeHooks.java:1105) [ForgeHooks.class:?]
	at net.minecraft.entity.player.EntityPlayer.interactOn(EntityPlayer.java:1294) [EntityPlayer.class:?]
	at net.minecraft.network.NetHandlerPlayServer.processUseEntity(NetHandlerPlayServer.java:1121) [NetHandlerPlayServer.class:?]
	at net.minecraft.network.play.client.CPacketUseEntity.processPacket(CPacketUseEntity.java:94) [CPacketUseEntity.class:?]
	at net.minecraft.network.play.client.CPacketUseEntity.processPacket(CPacketUseEntity.java:15) [CPacketUseEntity.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:21) [PacketThreadUtil$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_172]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_172]
	at net.minecraft.util.Util.runTask(Util.java:53) [Util.class:?]
	at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:798) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:743) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:192) [IntegratedServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:592) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_172]
```

**Comment by liach** [comment]
FYI, this issue was discovered in debugging for 493913f5e9326a9b118e6140a6e43451cfe4153e

**Comment by liach** [comment]
Fixed in 517f3d0cef8bd1b77287eb4777e61c09b949a52e
